### PR TITLE
MARS-194: Fixed issues related to cookies and setting optimizely attribute values

### PR DESCRIPTION
**Pull request by edwinf-barreram**
# Description
- Added some updates to send data to Optimizely API correctly

# Ticket
[MARS-194](https://kiva.atlassian.net/browse/MARS-194)

### CODE REVIEW

File: src/pages/Thanks/ThanksPage.vue [hunk 1]
```diff
@@ -288,9 +291,19 @@ export default {
...
-		const depositTotal = parseFloat(this.receipt?.totals?.depositTotals?.depositTotal);
-		userHasLentBefore(this.loans.length > 0);
-		userHasDepositBefore(depositTotal > 0);
+		const depositTotal = this.receipt?.totals?.depositTotals?.depositTotal;
+
+		const hasLentBefore = this.loans.length > 0;
+		const hasDepositBefore = parseFloat(depositTotal) > 0;
+
+		this.cookieStore.remove(hasLentBeforeCookie);
+		this.cookieStore.remove(hasDepositBeforeCookie);
```
Comment (dyersituations) [inline src/pages/Thanks/ThanksPage.vue:300]: In this two components, why does the cookie need to be removed first? To update the expiration date?

Comment (edwinf-barreram) [reply src/pages/Thanks/ThanksPage.vue]: Removed!

**Review by mcstover — APPROVED** [review]
I think this is generally fine, hopefully you can verify the behavior you expect in dev.

**Review by dyersituations — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
👍 good fix!

**Pull request merged**